src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+spring.datasource.url=jdbc:postgresql://${DB_HOST:localhost}:5432/competitionservice
```

Comment:
Yaml format is more like a standard in the industry, so let's use it.

Reply:
converted to yml